uber/configspec.ini

```diff
@@ -535,6 +535,16 @@ paid_by_group = string(default="paid by group")
 lost_badge    = string(default="lost badge")
 __many__ = string
 
+[[ledger_type]]
+accounts_received = string(default="Received")
+accounts_paid = string(default="Paid")
+
+[[ledger_method]]
+cc_stripe = string(default="Stripe")
+cc_reader = string(default="Credit Card Reader (e.g., Square)")
+cash = string(default="Cash")
+check = string(default="Check")
+
```

Comment:
FWIW, this approach does allow an event to just add Paypal in their config file without any code changes needed.